scania-digital-design-system/tegel
# chore(link): use singular Capital case where possible

**Describe pull-request**  
Updated comments and docs to use singular capital case.

**Solving issue**  
Part of: https://tegel.atlassian.net/browse/DTS-1329

**How to test**  
Go to storybook
Check in Components -> Link
Make sure it is always referenced with singular capital case (Link)
